### PR TITLE
{tools}[foss/2018b] twine v1.13.0 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2431.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/26580e9417717de068ab46c0aae7a3c5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2016.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c4b3af3f2c371917b9383269f573e620 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3118.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/5ba7b76afbbcafc2eab6fe285a61851e for a full test report.

File: easybuild/easyconfigs/t/twine/twine-1.13.0-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://twine.readthedocs.io'
+description = "Twine is a utility for publishing Python packages on PyPI."
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [('Python', '3.6.6')]
+
+#use_pip = True
```
Comment (akesandgren) [inline easybuild/easyconfigs/t/twine/twine-1.13.0-foss-2018b-Python-3.6.6.eb]: Travis complains on this!

Comment (boegel) [reply easybuild/easyconfigs/t/twine/twine-1.13.0-foss-2018b-Python-3.6.6.eb:14]: There was a reason I had to use this, but I don't recall what it was...

**Comment by boegel** [comment]
Not going to pursue this, since it's using a now deprecated toolchain version, so closing

**Pull request closed without merging**